cp23/hpi-fun2/motivation/executables/generator.py

```diff
@@ -0,0 +1,228 @@
+from pathlib import Path
+import random
+
+INPUT_LIMIT = 100_000
+
+SAMPLES = [
+["sample1", """11 20
+2 5 3 7 2 1 10 4 1 8 2
+1 2 2 2 5 5 1 8 8 8"""],
+["sample2", """5 7
+1 5 10 -2 -2
+1 1 3 3"""],
+["sample3", """5 7
+8 5 10 -2 -2
+1 1 3 3"""]
+]
+
+Path("sample1.in").write_text(SAMPLES[0][1])
+Path("sample1.desc").write_text("Basic test case; no unreachable nodes.")
```

Comment:
Now "sample1" is unused in `SAMPLES`. You may just want to do a list of 2-tuples (testcase, description) and create sample#i with a for loop. (just a suggestion)

Reply:
Done 👍🏼